packages/front-end/services/metrics.tsx

```diff
@@ -23,6 +23,13 @@ export function getMetricConversionTitle(type: MetricType): string {
   return "Conversion Rate";
 }
 
+export function getPercentileLabel(quantile: number): string {
+  if (quantile === 0.5) {
+    return "Median";
+  }
+  return `P${quantile * 100}`;
```

Comment:
Fun javascript trivia: Guess what `0.007 * 100` equals? 
 Wrong, it's `0.7000000000000001`.

Or another fun one: `0.974 * 100` equals `97.39999999999999`

Reply:
lmao ok I can truncate to one decimal.